start_stop.lua

```diff
@@ -472,6 +456,11 @@ function courseplay:start(self)
 		-- Initialize pure pursuit controller
 		self.cp.ppc = PurePursuitController(self)
 		self.cp.ppc:initialize()
+		if self.cp.driver ~= nil and self.cp.driver.collisionDetector ~= nil then
```

Comment:
Again, we are accessing here AI Driver internals. AI Driver should hide all these and do whatever it needs to do when is being started.

Reply:
Changed to self.cp.driver:delete()